cmd/govim/cpprotocol.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+// TODO: perhaps all this child-parent stuff is better housed in an internal helper
```

Comment:
agree

Reply:
Ok, will move as part of this PR.

Reply:
Actually, with your permission I will defer this until a later PR if that's ok?